scheduledtask-annotation/src/main/java/com/storebrand/scheduledtask/annotation/ScheduledTask.java

```diff
@@ -116,4 +124,11 @@
      * used to determine how many to keep.
      */
     int keepMaxFailedRuns() default 0;
+
+    /**
+     * Only keep this many noop runs. Older records will be deleted if there are more. This rule is set to 100 by
+     * default. If both this and {@link #keepMaxRuns()} is used then both will be applied, and the lowest number will be
+     * used to determine how many to keep.
+     */
+    int keepMaxNoopRuns() default ScheduledTaskBuilder.DEFAULT_KEEP_MAX_NOOP_RUNS;
```

Comment:
default for all other keep max is that they are disabled. We only enable deleteafterdays by default.